specification/purview/data-plane/Azure.Purview.Catalog/preview/2020-12-01-preview/purviewcatalog.json

```diff
@@ -0,0 +1,6748 @@
+    },
+    "/atlas/v2/entity/guid/{guid}": {
+      "get": {
+        "tags": [
+          "EntityREST"
+        ],
+        "operationId": "Entity_GetById",
+        "x-ms-examples": {
+          "EntityREST_GetById": {
+            "$ref": "./examples/EntityREST_GetById.json"
+          }
+        },
+        "summary": "Get complete definition of an entity given its GUID.",
+        "description": "Get complete definition of an entity given its GUID.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/guid"
+          },
+          {
+            "$ref": "#/parameters/minExtInfo"
+          },
+          {
+            "$ref": "#/parameters/ignoreRelationships"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK.",
+            "schema": {
+              "$ref": "#/definitions/AtlasEntityWithExtInfo"
+            }
+          }
+        }
+      },
+      "put": {
+        "tags": [
+          "EntityREST"
+        ],
+        "summary": "Update entity partially - Create/Update entity attribute identified by its GUID.",
+        "description": "Update entity partially - create or update entity attribute identified by its GUID.\nSupports only primitive attribute type and entity references.\nIt does not support updating complex types like arrays, and maps.\nNull updates are not possible.",
+        "operationId": "Entity_PartialUpdateEntityAttrByGuid",
+        "x-ms-examples": {
+          "EntityREST_PartialUpdateEntityAttrByGuid": {
+            "$ref": "./examples/EntityREST_PartialUpdateEntityAttrByGuid.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/guid"
+          },
+          {
+            "name": "name",
+            "in": "query",
+            "description": "The name of the attribute.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "in": "body",
+            "name": "body",
+            "description": "The value of the attribute.",
+            "required": true,
+            "schema": {
+              "type": "object"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Updated with EntityMutationResponse.",
+            "schema": {
+              "$ref": "#/definitions/EntityMutationResponse"
+            }
+          }
+        }
+      },
+      "delete": {
+        "tags": [
+          "EntityREST"
+        ],
+        "summary": "Delete an entity identified by its GUID.",
+        "description": "Delete an entity identified by its GUID.",
+        "operationId": "Entity_DeleteByGuid",
+        "x-ms-examples": {
+          "EntityREST_DeleteByGuid": {
+            "$ref": "./examples/EntityREST_DeleteByGuid.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/guid"
+          }
+        ],
+        "responses": {
+          "200": {
```

Comment:
Does this actually need to return something, how do callers interact with this result?

Reply:
Yes. It will return the deleted entity.